draft-ietf-rtcweb-jsep.xml

```diff
@@ -2030,10 +2030,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <xref target="RFC3264"></xref>, Section 5.1.</t>
 
               <t>If this m= section is for media with configurable
-              frame sizes, e.g. audio, an "a=maxptime" line, indicating
-              the smallest of the maximum supported frame sizes out of
-              all codecs included above, as specified in
-              <xref target="RFC4566"></xref>, Section 6.</t>
+              durations of media per packet, e.g. audio, an
+              "a=maxptime" line, indicating the smallest, out of all
+              codecs included above, of the maximum amount of media,
+              expressed as time in milliseconds, that can be
+              encapsulated in each packet as specified in <xref
```

Comment:
This seems awkward. I would say

"indicating the maximum amount of media, specified in milliseconds, that can be encapsulated in each packet, as specified in [RFC4566] This value is set to the smallest of the maximum values across all the codecs included in the m= section"